IntrinsicCategories/gap/IntrinsicCategories.gi

```diff
@@ -699,14 +699,17 @@ InstallMethod( IntrinsifyObject,
                1 := o
                );
     
-    ObjectifyObjectForCAPWithAttributes( obj, C );
+    ObjectifyWithAttributes( obj, IC!.object_type,
+            CapCategory, IC );
+    
+    SetFilterObj( obj, ObjectFilter( IC ) );
```

Comment:
The `SetFilterObj`s (here and the one for morphisms below) should not be needed (but neither to they break anything).

Reply:
What is your preference? Keep or remove?

Reply:
Thanks, removed.